introduction_to_amazon_algorithms/ipinsights_login/ipinsights-tutorial.ipynb

```diff
@@ -52,25 +52,25 @@
     "\n",
```

Comment:
<p>Can  <code>num_flagged_cases</code> or <code>num_all_positives</code> be zero? If yes, the code can fail.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1711/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>